lib/modules/manager/runtime-version/index.ts

```diff
@@ -0,0 +1,15 @@
+import type { Category } from '../../../constants';
+import { DockerDatasource } from '../../datasource/docker';
+
+export { extractPackageFile } from './extract';
+
+export const displayName = 'Runtime Version';
+
+export const defaultConfig = {
+  fileMatch: ['^runtime.txt$'],
```

Comment:
```suggestion
  fileMatch: ['(^|/)runtime.txt$'],
```
Can this really be only at the root of repos?

Reply:
Heroku
> To specify a Python runtime, add a runtime.txt file to your app’s root directory that declares the exact version number to use.

Koyeb
> To specify the Python version to use, add a runtime.txt file in the root of your project directory with one of the versions mentioned above

CloudFoundry
> You can specify a version of the Python runtime by including it within a runtime.txt file. For example:

Reply:
I'm not fussed if we relax this though